designer/server/src/views/forms/library.njk

```diff
@@ -36,7 +36,7 @@
 
       {% set formUpdated %}
         <span class="app-display-until-desktop">Updated </span>{{ (form.draft.updatedAt if form.draft.updatedAt else "2024-04-09") | formatDate }}<br>
-        by {{ form.draft.updatedBy if form.draft.updatedBy else "Enrique Chase" }}
+        by {{ form.draft.updatedBy.displayName if form.draft.updatedBy else "Enrique Chase" }}
```

Comment:
Now these are getting longer, might be worth using the `default()` filter?

```suggestion
        by {{ form.draft.updatedBy.displayName | default("Enrique Chase", true) }}
```

Reply:
Resolved with ac0e839608f3a65b966d366edef24d3c7aa40402